tools/sql/handler.go

```diff
@@ -266,17 +266,12 @@ func ValidateConnectConfig(cfg *config.SQL, isDryRun bool) error {
 		cfg.DatabaseName = schema.DryrunDBName
 	}
 	if cfg.TLS != nil && cfg.TLS.Enabled {
-		enabledCaFile := cfg.TLS.CaFile != ""
 		enabledCertFile := cfg.TLS.CertFile != ""
 		enabledKeyFile := cfg.TLS.KeyFile != ""
 
 		if (enabledCertFile && !enabledKeyFile) || (!enabledCertFile && enabledKeyFile) {
 			return schema.NewConfigError("must have both CertFile and KeyFile set")
 		}
-
-		if !enabledCaFile && !enabledCertFile && !enabledKeyFile {
-			return schema.NewConfigError("must provide tls certs to use")
-		}
 	}
```

Comment:
could you simple just delete this validation?
when the connection is created, ca / cert / key info will be propagated, there is no need to check it here

Reply:
and this test case can be deleted (handlerTest.go)
```go
cfg.TLS = &auth.TLS{}
cfg.TLS.Enabled = true
s.NotNil(sql.ValidateConnectConfig(cfg, false))
s.NotNil(sql.ValidateConnectConfig(cfg, true))
```

Reply:
Updated as requested.

Reply:
seems that tests after above code also needs to be deleted